src/lib/builders/slider/create.ts

```diff
@@ -174,27 +186,39 @@ export const createSlider = (props?: CreateSliderProps) => {
 
 				const thumbValue = $value[index];
 				const thumbPosition = `${$position(thumbValue)}%`;
+
+				const style: Record<string, string> = {
+					position: 'absolute',
+				};
+
+				if ($orientation === 'horizontal' && $dir === 'rtl') {
+					style.right = thumbPosition;
+					style.translate = '50% 0';
+				} else if ($orientation === 'horizontal') {
+					style.left = thumbPosition;
+					style.translate = '-50% 0';
+				} else if ($dir === 'rtl') {
+					style.top = thumbPosition;
+					style.translate = '0 -50%';
+				} else {
+					style.bottom = thumbPosition;
+					style.translate = '0 50%';
+				}
```

Comment:
```suggestion
const style: Record<string, string> = {
  position: 'absolute',
};

if ($orientation === 'horizontal') {
  style.left = $dir === 'rtl' ? undefined : thumbPosition;
  style.right = $dir === 'rtl' ? thumbPosition : undefined;
  style.transform = $dir === 'rtl' ? 'translate(50%, 0)' : 'translate(-50%, 0)';
} else {
  style.top = $dir === 'rtl' ? thumbPosition : undefined;
  style.bottom = $dir === 'rtl' ? undefined : thumbPosition;
  style.transform = $dir === 'rtl' ? 'translate(0, -50%)' : 'translate(0, 50%)';
}
```

Now that I think about it, why would rtl switch vertical directions?

Reply:
I read the comments now, ignore me :)